File: javascript/gpt/p124.js
const args = process.argv.slice(2);

if (args.length !== 1) {
    console.error("Error: Exactly one argument required.");
    process.exit(1);
}

let input = args[0];
console.log(`Original: '${input}'`);

input = input.replace(/\s+$/, '');
console.log(`Trimmed: '${input}'`);

process.exit(0);

File: java/gpt/p124.java
public class Main {
    public static void main(String[] args) {
        if (args.length != 1) {
            System.err.println("Error: Exactly one argument required.");
            System.exit(1);
        }

        String input = args[0];
        System.out.println("Original: '" + input + "'");

        input = input.replaceAll("\\s+$", "");
        System.out.println("Trimmed: '" + input + "'");

        System.exit(0);
    }
}
